### PR TITLE
Blockquote

### PR DESCRIPTION
**Pull request by prateksha**
#847 Resolved.

### CODE REVIEW

File: client/app/stories/Blockquote.stories.jsx [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+
+import Blockquote from 'bundles/shared/components/Blockquote';
+import { SingleColumnLayout, withSource } from './Stories.helper';
+
+storiesOf('Blockquotes', module)
+  .add('Blockquote', withSource(
+    <SingleColumnLayout>
+      <Blockquote label={'"It\'s not just all in your head, it\'s all around you. We can heal together."\u2764\uFE0F'} />
```
[the remark this answers is not in the document]
Comment (prateksha) [reply client/app/stories/Blockquote.stories.jsx]: Okay! I'll make the changes.
Does quotes generated with CSS mean that ,only the text is typed in stories.jsx and the quotes are automatically printed?

[the remark this answers is not in the document]
Comment (nshki) [reply client/app/stories/Blockquote.stories.jsx]: Yep that's right. The main text is passed as a prop and the quotes can be displayed with pseudo elements!

File: client/app/bundles/shared/components/Blockquote.scss [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  background-color: rgba(255,255,255,0.1);
+  font-family: Lato, sans-serif;
+  font-style: italic;
+  font-size: 36px;
+  line-height: 1;
+  text-align: center;
+  align-content: center;
+  text-shadow: 0px 2px 4px rgba(0, 0, 0, 0.25);
+
+  padding: 20px;
```
[the remark this answers is not in the document]
Comment (prateksha) [reply client/app/bundles/shared/components/Blockquote.scss]: Yes Okay!
So, what is the value of screen width for which I should set padding at 50px?

**Comment by prateksha** [comment]
@nshki I have made the changes. Please check if they are fine. :)

File: client/app/bundles/shared/components/Blockquote.scss [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+  align-content: center;
+  text-shadow: 0px 2px 4px rgba(0, 0, 0, 0.25);
+
+  width: 100%;
+  @media screen and (min-width: 1000px) {
+    padding: 50px;
+  }
+  @media screen and (max-width: 1000px) {
+    padding: 20px;
+  }
```
Comment (nshki) [inline client/app/bundles/shared/components/Blockquote.scss]: We can avoid using the `max-width` media query here if we just put the `padding: 20px;` above the `min-width` media query!

File: client/app/bundles/shared/components/Blockquote.jsx [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+  author?: string,
+};
+
+export default class Blockquote extends React.Component<Props> {
+  render() {
+    const { text, author } = this.props;
+    const textClassNames = `${css.author} ${css.text}`;
+    const authorClassNames = `${css.author}`;
+    return (
+      <div>
```
Comment (nshki) [inline client/app/bundles/shared/components/Blockquote.jsx]: Let's give the container a class here, and perhaps use a `<blockquote>`! All container properties should be applied to this class as well (`background-color`, `padding`, etc.). Currently, the quote is split into two parts, the text and author, which is great, but they each have container-like properties.

File: client/app/bundles/shared/components/Blockquote.jsx [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+        <div className={textClassNames}>
```
Comment (nshki) [inline client/app/bundles/shared/components/Blockquote.jsx]: Let's change this element to be a `<p>`.

**Comment by prateksha** [comment]
@nshki I have made the changes! For quotations, I have used <q> tag. Is it okay now?

**Comment by prateksha** [comment]
@nshki I have made the changes! I have used the <q> tag. Is it okay now?

File: client/app/bundles/shared/components/Blockquote.scss [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+@import "_base.scss";
+
+.text {
+  display: inline-block;
+  position: relative;
+
+  background-color: rgba(255, 255, 255, 0.1);
+  font-family: Lato, sans-serif;
+  font-style: italic;
+  font-size: 36px;
```
Comment (nshki) [inline client/app/bundles/shared/components/Blockquote.scss]: Let's make the default font size 24px and bump it up to 36px on larger screens -- sorry for missing this the first time!

File: client/app/bundles/shared/components/Blockquote.jsx [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+    const { text, author } = this.props;
+    const textClassNames = `${css.text}`;
+    return (
+      <div className={textClassNames}>
+        <p>
+          <q>
+            {text}
+          </q>
+        </p>
+        <div>
```
Comment (nshki) [inline client/app/bundles/shared/components/Blockquote.jsx]: Let's make sure there's a 10px gap between the quote text and author.

**Comment by prateksha** [comment]
Thanks @nshki !! Does everything seem fine now?

**Comment by nshki** [comment]
@prateksha Looks good! If we could do the author margin with a CSS class that'd be great. :)

**Comment by prateksha** [comment]
I created the CSS class. 
But after rebasing with master, circleci tests  are failing. How do I fix it?

**Comment by baohouse** [comment]
@prateksha It's some weird timeout issue. I'm reinitiating the test build.

**Comment by prateksha** [comment]
@baohouse It is still failing the circleci tests. This time the issue seems to be different.

**Pull request merged**